mgmtfn/k8splugin/contivk8s/k8s_cni.go

```diff
@@ -95,15 +279,40 @@ func addPodToContiv(nc *clients.NWClient, pInfo *cniapi.CNIPodAttr) {
 		}
 		os.Exit(1)
 	}
+	log.Infof("EP created IP: %s\n", result.Attr.IPAddress)
```

Comment:
Why change this function?
Why not change the 
// addPod is the handler for pod additions
func addPod(w http.ResponseWriter, r *http.Request, vars map[string]string) (interface{}, error) 

in \netplugin\mgmtfn\k8splugin\driver.go

Reply:
When, the runtime send the pid process the operation inside the container is permitted, but when is the path of network namespace I have problems to exec same operation I get the error:  `RTNETLINK answers: Invalid argument, exit status 2`

`ls -lha /var/run/netns/`
```
-rw-r--r--  1 root root    0 Aug  9 18:41 cni-d5e070e5-08c4-46f0-96ee-4bb5e6c6e21a
lrwxrwxrwx  1 root root   55 Aug  9 18:41 k8s_contiv-blue-c1_default_01bc671d-9a69-11e8-a90e-000af70485d0_1-81a93eff -> /var/run/netns/cni-d5e070e5-08c4-46f0-96ee-4bb5e6c6e21a
```

So, the k8s_cni is part of the contivk8s plugin execute on the host. 

I make some test to tray maintain the logical on `./netplugin\mgmtfn\k8splugin\driver.go` but no success.